src/main/java/com/google/collegeplanner/servlets/PlannerPageServlet.java

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 Google LLC
```

Comment:
missing test for this file.

Reply:
I wrote a test that tests whether each function is called, but I'm having trouble of thinking of how else to test it, because you forward the request and response to a black box servlet generated by jsp. Any ideas?

Reply:
I think what you have now works well.